about.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+
+    <!-- fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700|Gothic A1" rel="stylesheet" type="text/css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"
+        integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/fontawesome.min.css"
+        integrity="sha512-siarrzI1u3pCqFG2LEzi87McrBmq6Tp7juVsdmGY1Dr8Saw+ZBAzDzrGwX3vgxX1NkioYNCFOVC0GpDPss10zQ=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <!-- style sheet  -->
+    <link rel="stylesheet" href="./css/about.css" />
+</head>
+ 
+<body>
+    <section class="container">
+        <header>
+            <nav class="socialnavbar">
+                <ul class="socialnav">
+                    <li>
+                        <a href="/"><i class="fa-brands fa-facebook-f"></i></a>
+                    </li>
+                    <li>
+                        <a href="/"><i class="fa-brands fa-twitter"></i></a>
+                    </li>
+                    <li><a href="/">english</a></li>
+                    <li><a href="/">my page</a></li>
+                    <li><a href="/">logout</a></li>
+                </ul>
+            </nav>
+
+            <nav class="navbar">
```

Comment:
Ensure to hook up the js with this file as the hamburger is not working on the about page in order to deliver the same UX on all pages

Reply:
Hamburger is working on about page but the socialnav is not on figma design